package.json

```diff
@@ -19,11 +19,12 @@
     "start": "ember serve",
     "test": "ember test",
     "test:all": "ember try:each",
-    "test:node": "mocha node-tests/**"
+    "test:node": "for i in node-tests/*/*; do mocha $i; done"
```

Comment:
There seems to be strange issues with `ember-cli-addon-tests` triggered only if multiple tests are executed at the same time and also depending on the order in which the tests are executed. While this is definitely not a good solution it gets us unblocked here.

Reply:
Reported this upstream: https://github.com/tomdale/ember-cli-addon-tests/issues/215

Reply:
Eeck, seems fine but that sounds like it was pretty gnarly to track down.